README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
+## Build Instructions 
+
+BUILD INSTRUCTIONS
+
+◾️Introduction
+
+This documents intent is to support developers in understanding how to 
+build the Image Lab project. This document offers support on key useful commands and error solutions that can make building Image Lab a smoother process.
+
+◾️Before You Build
+- Install Gradle
+ sudo apt install gradle
+- Install Canberra
+ Sudo apt install libcanberra-gtk-module libcanberra-gtk3-module
+
+
+◾️What is Gradle
+
+Gradle is a automated build system that makes working with large projects easier.
+It streams line the build process by skipping re-execution and allows developers to preform tasks suing one line of code in the command line.
+
+
+
+◾️How does Gradle Work?
+
+▪️Project 
+- A project is a depiction of what needs to be built and how we get an artifact at the end of a build.
+
+settings.gradle - Where projects are registered.
+build.gradle - constructs commands that can be used to execute tasks.
```

Comment:
This is expressed in jargon that someone needs to already be familiar with in order to understand.  (That person probably already know where Gradle stores this information.)